schema/crdb/inventory-omicron-sled-config/up10.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE omicron.public.inv_sled_agent
+    ADD COLUMN IF NOT EXISTS reconciler_status_kind inv_config_reconciler_status_kind NOT NULL;
```

Comment:
Does this need to set a default value (not-yet-run?) in this migration, and then remove the default in a future one?

Reply:
I was worried it might, but it doesn't, because we've explicitly removed all the rows from `inv_sled_agent` in `up01.sql`.